test/unit/matrix/test_util_matrix.cpp

```diff
@@ -132,7 +139,7 @@ void check_is_hermitian(Matrix<const T, Device::CPU>& matrix, comm::Communicator
         continue;
 
       if (current_rank == owner_original) {
-        const auto& tile_original = matrix.read(index_tile_original).get();
+        const auto& tile_original = sync_wait(matrix.read_sender(index_tile_original)).get();
```

Comment:
Exactly correct. The extra `.get` never actually waits for the future in this situation; it will always be ready. But it's needed for unwrapping. This is something that needs to be cleaned up. This situation doesn't actually seem to come up in the library itself, but I wanted to keep the test as close as possible to what it used to be.